src/ocamlorg_frontend/pages/install.eml

```diff
@@ -0,0 +1,191 @@
+              and any packages you install.
+              This command installs OCaml 5.0.0 in a global <code>opam</code> switch:
+            </p>
+
+            <%s! Copy_to_clipboard_snippet.render ~id:"create-opam-switch" "opam switch create 5.0.0" %>
+
+            <p>
+              <code>opam switch list-available</code> lists all available compiler versions.
+            </p>
+
+          </li>
+        </ol>
+
+        <p>Congratulations, you have installed OCaml!</p>
+      </div>
+      <div>
+        <div class="prose">
+          <h2>Set Up an OCaml Development Environment</h2>
+
+          <p>
+            The <a href="<%s Url.tutorial "up-and-running" %>#installing-the-ocaml-platform-tools">OCaml Platform Tools</a>
+            complete your OCaml development environment. To install them in your current <code>opam</code> switch, run this command:
+          </p>
+
+          <%s! Copy_to_clipboard_snippet.render ~id:"install-platform-tools" "opam install dune merlin ocaml-lsp-server odoc ocamlformat utop dune-release" %>
+
+          <p>Now you are ready to write some OCaml code!</p>
+        </div>
+        <a class="btn text-white mt-6 flex gap-2" href="<%s Url.tutorial "up-and-running" %>#first-steps-with-ocaml">First Steps with OCaml <%s! Icons.arrow_small_right "h-6 w-6" %></a>
+
+        <div class="prose mt-6">
+          <p>If the instructions on this page did not work for you, you can find
+            <a href="<%s Url.getting_started %>">a more detailed explanation of OCaml installation methods here</a>.</p>
+        </div>
+      </div>
+    </div>
+
+    <div class="grid gap-6 xl:grid-cols-2 xl:gap-16" x-show="operating_system == 'win'">
+      <div class="prose">
+        <h2>
+          Install OCaml on Windows
+        </h2>
+        <ol>
+          <li>
+            <h3>Use the Diskuv OCaml ("DKML") Windows installer</h3>
+              <p>
+                The installer sets up OCaml 4.14.0 with Git and the Visual Studio compiler.
+              </p>
+
+              <a class="btn !text-white hover:no-underline" href="https://github.com/diskuv/dkml-installer-ocaml/releases/download/v1.1.0_r2/setup-diskuv-ocaml-windows_x86_64-1.1.0.exe">
+                <%s! Icons.download "h-6 w-6" %> Download DKML Installer
+              </a>
+
+              <p>
+                Currently, 4.14.0 is the only OCaml version available through the Diskuv OCaml Installer.
+                If you need a different OCaml version, check out the alternative installation instructions
+                provided in <a href="<%s Url.ocaml_on_windows %>">"OCaml on Windows"</a>.
+              </p>
+          </li>
+          <li>
+            <h3>Create an <code>opam</code> switch</h3>
+
+            <p>
+              An <code>opam</code> switch (named after
+              <a href="https://github.com/ocaml/opam">OCaml's package manager <tt>opam</tt></a>)
```

Comment:
```suggestion
              OCaml's package manager  <a href="https://github.com/ocaml/opam"><tt>opam</tt></a>)
```
This looks a bit awkward otherwise due to the shifting font:
<img width="536" alt="image" src="https://user-images.githubusercontent.com/53164/231194301-ab920502-2c63-4178-82f7-77dc412e6793.png">